docs/content/guides/dagster/branch_deployments.mdx

```diff
@@ -20,7 +20,7 @@ Here’s an overview of the main concepts we’ll be using:
 - [Ops](/concepts/ops-jobs-graphs/ops) - We'll define two ops that query Snowflake: the first will clone a database, and the second will drop database clones.
 - [Graphs](/concepts/ops-jobs-graphs/graphs) - We'll build graphs that define the order our ops should run.
 - [Jobs](/concepts/assets/asset-jobs) - We'll define jobs by binding our graphs to resources.
-- [Resources](/concepts/resources) - We'll use the [Snowflake resource](/\_apidocs/libraries/dagster-snowflake#dagster_snowflake.snowflake_resource) to swap in different Snowflake connections to our jobs depending on environment.
+- [Resources](/concepts/resources) - We'll use the [Snowflake resource](/\_apidocs/libraries/dagster-snowflake#dagster_snowflake.SnowflakeResource) to swap in different Snowflake connections to our jobs depending on environment.
```

Comment:
This guy works with PyObject - lets use that so we don't need to update this link again:

```suggestion
- [Resources](/concepts/resources) - We'll use the Snowflake resource (<PyObject object="SnowflakeResource" module="dagster_snowflake" />) to swap in different Snowflake connections to our jobs depending on the environment.
```